### ISSUE TITLE
Timer / stopwatch / alarm skills

### ISSUE THREAD

**Issue by Trial-In-Error** [stanford-oval/thingpedia-common-devices#345]
Does almond have any 1st/3rd party time-aware skills? The web interface doesn't recognize any of my attempts to trigger these behaviors.

**Comment by gcampax** [comment]
Almond has basic support for timer as a primitive available in every skill: any command can be repeated at a given time every day, or at a fixed interval. The wording is something like "every day at 7pm turn on the lights". If these commands do not work, it is a bug.

Simple reminders should also work, but we're working to make them more robust and more useful, especially one-off reminders.

**Comment by Trial-In-Error** [comment]
I'm not currently at a computer w/almond-server installed, so I'm trying various things via Almond's [try it out](https://dev.almond.stanford.edu/) interface. Some use-cases work fairly well (maybe the first 50% or so?). Here are some example (good and bad) interactions. Since I haven't configured an account, I can't actually tell if these are working as intended or throwing some of the input out on parsing, but:

```
# BAD: short relative time (e.g., for timers) isn't recognized
> in 5 minutes, log a message "hello world"
Sorry, I did not understand that. Can you rephrase it?

> 5 minutes from now, log a message "hello world"
Sorry, I did not understand that. Can you rephrase it?

# GOOD: absolute time (e.g., for timers or alarms or notifications) is recognized
> at 10:23, log a message "hello world"
To receive notifications you must first create a personal Almond account.

# GOOD: long relative time (e.g., for timers) is recognized
> in an hour, log a message "hello world"
To receive notifications you must first create a personal Almond account.

> in 90 minutes, log a message "hello world"
To receive notifications you must first create a personal Almond account.

# BAD: I can't find any way to start a stopwatch / duration
> start a stopwatch
Sorry, I did not understand that. Can you rephrase it?

# BAD: some attempts to start a stopwatch erroneously report the current time
> start counting time
Right now, it's 10:22 AM.


```

**Comment by gcampax** [comment]
Thanks for trying things. I would suggest trying from inside the logged in interface of Web Almond because that will show you if the agent actually runs the right timer or not.

Anyway, tagging @emilyjchang who is working on timers at the moment.

**Comment by tonyespinoza1** [comment]
the relative timer seems to be working now (go emily!) 

i am filing as enhancement, P3 for the timer/stop watch functionality we should build down the road.
